src/pt/pizzariascan/src/eu/kanade/tachiyomi/extension/pt/pizzariascan/Pizzariascan.kt

```diff
@@ -10,5 +13,15 @@ class Pizzariascan : Madara(
     "pt-BR",
     SimpleDateFormat("dd 'de' MMMM 'de' yyyy", Locale("pt")),
 ) {
+    override val supportsLatest = false
```

Comment:
All the filter options in Madara seem to work. The website seem kinda messed up in that the actual 'latest' list is labeled as 'popular'.

These all produce different results:
```
$baseUrl/manga/?m_orderby=latest
$baseUrl/manga/?m_orderby=alphabet
$baseUrl/manga/?m_orderby=rating
$baseUrl/manga/?m_orderby=trending
$baseUrl/manga/?m_orderby=views
$baseUrl/manga/?m_orderby=new-manga
```